crashreporter/src/main/java/com/tealium/crashreporter/CrashReporter.kt

```diff
@@ -0,0 +1,41 @@
+package com.tealium.crashreporter
+
+import com.tealium.crashreporter.CrashReporter.Tracker
+import com.tealium.internal.data.Dispatch
+import com.tealium.internal.listeners.ActivityResumeListener
+import com.tealium.internal.listeners.DisableListener
+import com.tealium.internal.listeners.PopulateDispatchListener
+import com.tealium.library.DataSources
+import com.tealium.library.Tealium
+
+/**
+ * Crash Reporter module for tracking crash data. When used, this module populates
+ * <i>crash_*</i> attributes defined in {@link com.tealium.library.DataSources.Key}.
+ */
+class CrashReporter (config: Tealium.Config, tracker: Tracker, truncateStackTrace: Boolean) : Thread.UncaughtExceptionHandler {
```

Comment:
CrashReporter should conform to either a Core Collector or Module. Take a look at the other modules for reference. 
`Tealium.Config` does not exist in `tealium-kotlin`. Should be using `TealiumConfig` and can likely be private.
`tracker: Tracker` can be removed, as you won't be needing it. 
Not 100% on this, but I suspect we'll need to incorporate [CoroutineExceptionHandler](https://kotlinlang.org/docs/reference/coroutines/exception-handling.html#exception-handling)